jenkins/opensearch/integ-test.jenkinsfile

```diff
@@ -30,12 +30,6 @@ pipeline {
         BUILD_JOB_NAME = "distribution-build-opensearch"
         ARTIFACT_BUCKET_NAME = credentials('jenkins-artifact-bucket-name')
     }
-    triggers {
-        parameterizedCron '''
-            H 3 * * * %TEST_MANIFEST=2.9.0/opensearch-2.9.0-seg-rep-test.yml;BUILD_MANIFEST_URL=https://ci.opensearch.org/ci/dbc/distribution-build-opensearch/2.9.0/latest/linux/arm64/tar/builds/opensearch/manifest.yml
-            H 3 * * * %TEST_MANIFEST=2.9.0/opensearch-2.9.0-seg-rep-test.yml;BUILD_MANIFEST_URL=https://ci.opensearch.org/ci/dbc/distribution-build-opensearch/2.9.0/latest/linux/x64/tar/builds/opensearch/manifest.yml
-            '''
-    }
```

Comment:
qq, for my understanding, why revert these?

Reply:
We are close to finish the `2.9.0` and finalize the RC, at this point we exclude the release from running periodically.
More details: https://github.com/opensearch-project/opensearch-build/blob/120c469e493142f1b045691d43548d35c6544d7c/RELEASE_PROCESS_OPENSEARCH.md#release-candidate-lock